lldb/source/Target/Process.cpp

```diff
@@ -6544,7 +6545,8 @@ static void AddRegion(const MemoryRegionInfo &region, bool try_dirty_pages,
     return;
   if (try_dirty_pages && AddDirtyPages(region, ranges))
     return;
-  ranges.push_back(CreateCoreFileMemoryRange(region));
+
+  ranges.Append(region.GetRange().GetRangeBase(), region.GetRange().GetByteSize(), CreateCoreFileMemoryRange(region));
```

Comment:
format, line too long?

Reply:
Currently I'm having issues with my git-clang format. I'll sort before we go to commit.